server.js

```diff
@@ -123,6 +123,21 @@ app.get(/\.html?$/i, function (req, res){
   res.redirect(path);
 });
 
+// redirect if `redirect(<URL>)` is part of the submitted data.
+app.use('*',function(req,res,next)
```

Comment:
Our existing functions have more whitespace - eg spaces between params. should we match?